pkg/cmd/cli/schedule/create.go

```diff
@@ -93,6 +93,9 @@ func NewCreateOptions() *CreateOptions {
 
 func (o *CreateOptions) BindFlags(flags *pflag.FlagSet) {
 	o.BackupOptions.BindFlags(flags)
+	// hide the Backup options --from-schedule flag
+	// TODO: see if there is a way to remove the flag
+	flags.MarkHidden("from-schedule")
```

Comment:
One option would be to bind the `--from-schedule` flag in a different method than `BackupOptions.BindFlags(...)`, and to *only* call that new method in the backup create code. Take a look at `BindWait(...)` in the backup create code.

Reply:
Ah thanks for the pointer, that sounds like a much better approach!